### PR TITLE
Update run_data_structures.py

### PR DESCRIPTION
**Pull request by atokuz**
Imbalance_array length corrected

# Description

In run_data_structures _get_expected_imbalance function 
`len(imbalance_array) 
`
 always return 2.

Subscript should have been added as
`ewma_window = int(min(len(imbalance_array['buy']), window))`

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x ] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [ x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: mlfinlab/data_structures/run_data_structures.py [hunk 1]
```diff
@@ -197,11 +197,10 @@ def _get_expected_imbalance(self, window, imbalance_array):
                 imbalance_array['buy'][-ewma_window:], dtype=float)
             sell_sample = np.array(
                 imbalance_array['sell'][-ewma_window:], dtype=float)
-            buy_and_sell_imb = sum(buy_sample) + sum(sell_sample)
```
Comment (Jackal08) [inline mlfinlab/data_structures/run_data_structures.py]: @proskurin have you seen these changes since your last approval?

Comment (proskurin) [reply mlfinlab/data_structures/run_data_structures.py]: Sorry, didn't see that. @atokuz for proportion we need to divide by buy_and_sell_imb, by simply taking buy_sample we don't get proportion

File: mlfinlab/data_structures/run_data_structures.py [hunk 2]
```diff
@@ -197,11 +197,10 @@ def _get_expected_imbalance(self, window, imbalance_array):
...
             exp_buy_proportion = ewma(
```
Comment (proskurin) [inline mlfinlab/data_structures/run_data_structures.py]: this expression doesn't give is proportion, it gives expected buy imbalance instead

**Comment by atokuz** [comment]
run_bar implementation in dev branch resolve these issues.

**Comment by Jackal08** [comment]
@proskurin I see this is closed. Is there anything in this branch that should be in dev?

**Comment by Jackal08** [comment]
> run_bar implementation in dev branch resolve these issues.

Ah yes, we always work on the dev branch. I am the only one with permissions to push to master.

**Comment by proskurin** [comment]
Still min(len(imbalance_array['buy']), window) change is a valid change

**Comment by proskurin** [comment]
Please create PR with merge into dev

**Comment by atokuz** [comment]
> Please create PR with merge into dev

Sorry about that. I am still trying to learn my way around github.

**Comment by Jackal08** [comment]
Lets get this branch passing the build server checks, then I will review.

**Pull request closed without merging**